GpsDataCapturer/mobile/src/main/java/com/google/gpsdatacapturer/MobileGpsMainActivity.java

```diff
@@ -174,7 +174,7 @@ public void onStopButtonClick(View view) {
             Log.e(TAG, "GpsDataCaptureService is not bound");
         }
         Log.d(TAG, "Stop capture data.");
-        gpsDataCaptureService.stopCapture();
+        gpsDataCaptureService.stopCapture(apiVersion);
```

Comment:
apiVersion is confusing name; usually version is an integer identifying some incremental changes.
here is probably apiChoice? or apiSelector or anything but version.